account_banking_ach_direct_debit/tests/test_account_payment.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2021 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from datetime import datetime
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPayment(TransactionCase):
+    def setUp(self):
+        super(TestPayment, self).setUp()
```

Comment:
```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
```

If there's a test class using `setUp` to generate test records, move it to `setUpClass` as it's way more performant since it runs only once. see: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0

Reply:
@xaviedoanhduy @JordiBForgeFlow  can we resolve or make a new PR here?

Reply:
Updated, Thanks